_projects/2022/100383017/100383017.Rmd

```diff
@@ -0,0 +1,634 @@
+---
+title: "Country Convergence"
+description: "This project enables the user to recreate a specific graph in OurWorldinData on GDP per capita layer by layer. It also introduces two alternative representations resorting to the same data."
+categories: "2022"
+author: Nacho Pulido Ruiz
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: TRUE
+---
+
+## On the topic of Economics and Country Convergence
+
+Our aim will be to resort to visual tools in R studio and ggplot to study the topic of country economic growth and convergence. Classical Solow-Swan models of growth predict convergence as a function of the deviation of a country from its stationary state (natural level) such that we should expect convergence between LEDC´S and MEDC´s. Our point of departure will be to launch the libraries we will use and to obtain the data from the Maddison Project Database.
+
+Original plot:
+
+[![](images/original_plot.png){width="500"}](https://ourworldindata.org/economic-growth)
+
+```{r setup}
+knitr::opts_chunk$set(fig.asp = 0.7,
+                      fig.align = "center")
```

Comment:
It may be good to set `out.width="100%"` too globally? Or individually in some chunks where images are a bit small when you set fixed coordinates.